modules/composer-workflowoperation/src/main/java/org/opencastproject/workflow/handler/composer/ImageWorkflowOperationHandler.java

```diff
@@ -176,6 +179,10 @@ WorkflowOperationResult main(final MediaPackage mp) throws WorkflowOperationExce
         logger.info("No source tracks found in media package {}, skipping operation", mp.getIdentifier());
         return handler.createResult(mp, Action.SKIP);
       }
+      if (cfg.skipIfTargetFlavorHasElements && mp.getElementsByFlavor(cfg.targetImageFlavor).length > 0) {
```

Comment:
Couldn't the target flavor be `*/*`? In which case it would end up being the source flavor? This check wouldn't catch that, would it?